### PR TITLE
updated steps

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/476

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/enterprise/identity-access-policies.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ Create a policy for each platform:
 - Android
 - Windows 10
 
-To create a new app protection policy, log in to the Microsoft Azure portal with your administer credentials, and then navigate to **Mobile apps > App protection policies**. Choose **Add a policy**.
+To create a new app protection policy, log in to the Microsoft Azure portal with your administer credentials, and then navigate to **Client apps > App protection policies**. Choose **Create policy**.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/identity-access-policies.md]:
```suggestion
To create a new app protection policy, sign in to the Microsoft Azure portal with your administrator credentials, and then navigate to **Client apps** > **App protection policies**. Choose **Create policy**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@VLG17 - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@BrendaCarter - PR has been copy edited and is ready for final review. This is based on issue #476

Thank you.

cc: @kenwith

**Pull request merged**